docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
-      { pathname: '/x/react-data-grid/overlays', title: 'Overlays' },
-      { pathname: '/x/react-data-grid/components', title: 'Custom subcomponents' },
-      {
-        pathname: '/x/react-data-grid/style-group',
-        title: 'Style',
-        children: [
-          { pathname: '/x/react-data-grid/style', title: 'Overview' },
-          { pathname: '/x/react-data-grid/style-recipes', title: 'Recipes' },
-        ],
-      },
-      { pathname: '/x/react-data-grid/localization' },
       { pathname: '/x/react-data-grid/scrolling' },
-      { pathname: '/x/react-data-grid/virtualization' },
-      { pathname: '/x/react-data-grid/accessibility' },
-      { pathname: '/x/react-data-grid/performance' },
-      { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
-      { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
-      { pathname: '/x/react-data-grid/aggregation', plan: 'premium' },
-      { pathname: '/x/react-data-grid/pivoting', plan: 'premium', planned: true },
+
       {
         pathname: '/x/react-data-grid/list-view',
         title: 'List view',
         plan: 'pro',
         unstable: true,
       },
       {
-        pathname: '/x/react-data-grid/server-side-data-group',
-        title: 'Server-side data',
-        plan: 'pro',
-        newFeature: true,
+        pathname: '/x/api/resources-group',
+        subheader: 'Resources',
         children: [
-          { pathname: '/x/react-data-grid/server-side-data', title: 'Overview', plan: 'pro' },
-          { pathname: '/x/react-data-grid/server-side-data/tree-data', plan: 'pro' },
-          {
-            pathname: '/x/react-data-grid/server-side-data/lazy-loading',
-            plan: 'pro',
-            planned: true,
-          },
           {
-            pathname: '/x/react-data-grid/server-side-data/infinite-loading',
+            pathname: '/x/react-data-grid/server-side-data-group',
+            title: 'Server-side data',
             plan: 'pro',
-            planned: true,
-          },
-          {
-            pathname: '/x/react-data-grid/server-side-data/row-grouping',
-            plan: 'premium',
-          },
-          {
-            pathname: '/x/react-data-grid/server-side-data/aggregation',
-            plan: 'premium',
-            planned: true,
+            children: [
+              { pathname: '/x/react-data-grid/server-side-data', title: 'Overview' },
+              { pathname: '/x/react-data-grid/server-side-data/tree-data', plan: 'pro' },
+              {
+                pathname: '/x/react-data-grid/server-side-data/lazy-loading',
+                plan: 'pro',
+                planned: true,
+              },
+              {
+                pathname: '/x/react-data-grid/server-side-data/infinite-loading',
+                plan: 'pro',
+                planned: true,
+              },
+              {
+                pathname: '/x/react-data-grid/server-side-data/row-grouping',
+                plan: 'pro',
```

Comment:
Row grouping is a Premium feature

Reply:
good catch! Not sure how that got changed 🤔